build_overrides/dawn.gni

```diff
@@ -7,6 +7,7 @@
 dawn_standalone = true
 
 dawn_glfw_dir = "//third_party/glfw"
+dawn_glm_dir = "//third_party/dawn/third_party/glm"
```

Comment:
Is glm required by Aquarium now? I think we just use the math functions in Matrix.h.

Reply:
No, but now we cannot build Dawn without defining this build override. See https://dawn-review.googlesource.com/c/dawn/+/19202, and maybe I should reword the commit message to include the offending CLs.